RAG_supporters/embeddings/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""
+Embedding utilities for keyword extraction, embedding creation, and processing.
```

Comment:
Remove this deprecated dependencies. Deprecation is not needed since those functions are not released yet